### ISSUE TITLE
OACMPDM4 missing texture

### ISSUE THREAD

**Issue by Bishop-333** [NeonKnightOA/oacmpvol1#34]
![shot0010](https://user-images.githubusercontent.com/99912467/162936040-04620311-b663-4f63-b80f-e4be13dac09b.jpg)

**Comment by EddieBrrrock** [comment]
all the walls around are extremely dark and we cannot even see if it's a wall or void

**Comment by EddieBrrrock** [comment]
z_oacmp1v4_beta-220411

**Comment by NeonKnightOA** [comment]
Indeed, lighting is a problem with this map. I'll be sure to take a look.

**Comment by The-Gig** [comment]
I don't remember the reason why you recompiled the map in the first place... maybe the original compiling by Akom had better lighting?

**Comment by NeonKnightOA** [comment]
It did, but only when compiled with Q3Radiant. When compiling with NetRadiant's q3map2 the map went dark.

**Comment by NeonKnightOA** [comment]
If Q3Radiant didn't generate non-GPL-compliant files, we wouldn't have to undergo 90% of the problems with this pack. \*sigh\*

Sometimes, as much as I love the spirit of the GPL license, I **really** hate the license itself.

**Comment by NeonKnightOA** [comment]
It's GTKRadiant the one that was released under GPL. Q3Radiant is still closed source.

**Comment by The-Gig** [comment]
Are you sure?
Here <https://github.com/id-Software/Quake-III-Arena/blob/master/README.txt> it mentions Q3Radiant and q3map (which are also the names of these folders <https://github.com/id-Software/Quake-III-Arena/tree/master/q3radiant> and <https://github.com/id-Software/Quake-III-Arena/tree/master/q3map>)...

**Comment by NeonKnightOA** [comment]
Took me a while but I found [leilei's comment on q3radiant](https://openarena.ws/board/index.php?topic=4679.msg47222#msg47222).

> Akom probably maps for OA using a Quake3 installation, which kind of goes against the whole this-isnt-q3 independence of the project.
> I believe I posted before on NOT recommending old q3radiant and q3map

Unfortunately I haven't found the previous not recommendation on q3radiant. But yeah, over time I grew up tired of GPLv2, its problems, the compliance and how in some situations it causes more headaches than solutions when it should be the other way around.

**Comment by NeonKnightOA** [comment]
I'll try recompiling this map without the -dark switch to see if that's the cause.

**Comment by NeonKnightOA** [comment]
https://mega.nz/file/bMZkRJBa#J_rUm2nz2SpiMFl6kBY0eMceeCLOJ1PLfnz_-CQRzCw
New test version. I've relit the whole map. Turns out the lights weren't shaders but textures, and the light in them was... quite dim. So I've created separate shaders for each light that seemed to solve the issue (still had to apply a light actor on top of them, though). **Only two lights missing.** Also separated the stars into a separate skybox.
